opendatahub-io/odh-dashboard
# Improve performance in nb-events

## Description

Partial fix of #627

## How Has This Been Tested?



1. Deploy odh-dashboard
2. Generate a new notebook
3. Check that nb-events only  are called when the notebook is being created AND we only fetch events for that specific notebook.

## Merge criteria:
